python/ray/experimental/state/common.py

```diff
@@ -49,3 +49,26 @@ class WorkerState:
     worker_id: str
     is_alive: str
     worker_type: str
+
+
+@dataclass(init=True)
+class TaskState:
+    task_id: str
+    name: str
+    scheduling_state: str
+
+
+@dataclass(init=True)
+class ObjectState:
+    object_ref: str
```

Comment:
I added a custom logic to change the key. The problem was `construct_memory_table` returns object_ref instead of object_id, and it is a little dangerous to change this behavior.